src/modules/modifyTask.js

```diff
@@ -0,0 +1,13 @@
+import { toNumber } from 'lodash';
+
+export default function modifyTask(newdescription, index /* completed = false */) {
```

Comment:
Please avoid pushing old code (commented out code) to Github.

Reply:
Solved 👍🏽  Thanks